fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -56,6 +61,13 @@ def self.show_details(filter: nil)
         print_output_variables(action, filter)
         print_return_value(action, filter)
 
+        if Fastlane::Actions.is_class_action?(action) && action.category == :deprecated
+          puts "==========================================".deprecated
```

Comment:
Ah, right, sorry about that. I'm a simple man, I see `puts` I say "Use `UI` instead".